apps/questionAndAnswer/src/view/progressReport/SearchResultsDrawer.tsx

```diff
@@ -9,20 +9,17 @@ import {
   VuiSpacer,
   VuiText,
   VuiTextColor,
-  VuiTitle,
-  truncateEnd,
-  truncateStart
+  VuiTitle
 } from "../../ui";
 import { useSearchContext } from "../../contexts/SearchContext";
 import "./searchResultsDrawer.scss";
+import { parseSnippet } from "../../utils/parseSnippet";
 
 type Props = {
   isOpen: boolean;
   onClose: () => void;
 };
 
-const CONTEXT_MAX_LENGTH = 200;
```

Comment:
We ran into cases where search results were super long, making it difficult to look through all of the results in the UI. So we implemented this limit to guard against that UX problem. I see this is removed here and in searchSummary app, but kept in search. Was this deliberate?

Reply:
Yes this was. Leaving it in search was accidental as I meant to take it out of there too.

The reason for taking this out is that keeping it in negates a lot of the sentences/chars before/after configuration. I ran into this problem in console, thinking it was a bug with query configs, but it was actually due to the display.

Then I ran into it here as well. I thought I had issues with how I was configuring the query, since the sentences/chars context wasn't being displayed. Then I remembered that we truncate.

IMO if a user sets these, they should be able to see the direct effect of it and deal with it as they please. WDYT?

Reply:
> IMO if a user sets these, they should be able to see the direct effect of it and deal with it as they please

I agree. I think the confusion you ran into is a valid problem and we should address it. At the same time, I think the original problem is still valid because the initial experience can be really poor, and having some truncation logic already in the codebase saves you some time from having to write it yourself. Is there a way we can solve both problems? Maybe we can make this variable easier to discover/configure and/or document it?